<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1730</width>
    <height>809</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="exitButton">
    <property name="geometry">
     <rect>
      <x>1590</x>
      <y>40</y>
      <width>88</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>Выход</string>
    </property>
   </widget>
   <widget class="QSpinBox" name="personForDeleteNumberSpin">
    <property name="geometry">
     <rect>
      <x>1460</x>
      <y>710</y>
      <width>41</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="deleteButton">
    <property name="geometry">
     <rect>
      <x>1510</x>
      <y>710</y>
      <width>131</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Удалить</string>
    </property>
   </widget>
   <widget class="QTableWidget" name="table">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>1241</width>
      <height>741</height>
     </rect>
    </property>
    <column>
     <property name="text">
      <string>ФИО</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Пол</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Возраст</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Рост</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Вес</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Возраст партнёра</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Рост партнёра</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Вес партнёра</string>
     </property>
    </column>
   </widget>
   <widget class="QRadioButton" name="addRadioButton">
    <property name="geometry">
     <rect>
      <x>1310</x>
      <y>60</y>
      <width>111</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Добавить</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_7">
    <property name="geometry">
     <rect>
      <x>1270</x>
      <y>10</y>
      <width>301</width>
      <height>17</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>14</pointsize>
      <weight>50</weight>
      <italic>true</italic>
      <bold>false</bold>
      <underline>false</underline>
      <strikeout>false</strikeout>
     </font>
    </property>
    <property name="text">
     <string>Вы желаете изменить человека</string>
    </property>
   </widget>
   <widget class="QWidget" name="gridLayoutWidget_2">
    <property name="geometry">
     <rect>
      <x>1270</x>
      <y>530</y>
      <width>411</width>
      <height>161</height>
     </rect>
    </property>
    <layout class="QGridLayout" name="gridLayout_2">
     <item row="5" column="0">
      <widget class="QCheckBox" name="sortCheckBox2">
       <property name="text">
        <string>Отсортировать по возрасту</string>
       </property>
      </widget>
     </item>
     <item row="3" column="1">
      <widget class="QLabel" name="label_4">
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item row="5" column="1">
      <widget class="QPushButton" name="findButton">
       <property name="text">
        <string>Найти</string>
       </property>
      </widget>
     </item>
     <item row="4" column="0">
      <widget class="QLabel" name="label_3">
       <property name="text">
        <string>Найти пару для человека №:</string>
       </property>
      </widget>
     </item>
     <item row="0" column="0">
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string>Вычислить возможные пары</string>
       </property>
      </widget>
     </item>
     <item row="4" column="1">
      <widget class="QSpinBox" name="personForFindNumberSpin"/>
     </item>
     <item row="0" column="1">
      <widget class="QRadioButton" name="forManRadioButton">
       <property name="text">
        <string>для мужчин</string>
       </property>
      </widget>
     </item>
     <item row="3" column="0">
      <widget class="QLabel" name="label_5">
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QRadioButton" name="forWomanRadioButton">
       <property name="text">
        <string>для женщин</string>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QPushButton" name="calcPosibblePairsButton">
       <property name="text">
        <string>Вычислить пары</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QLabel" name="label_8">
    <property name="geometry">
     <rect>
      <x>1320</x>
      <y>30</y>
      <width>221</width>
      <height>17</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>14</pointsize>
      <italic>true</italic>
     </font>
    </property>
    <property name="text">
     <string> или добавить нового?</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="changeRadioButton">
    <property name="geometry">
     <rect>
      <x>1440</x>
      <y>60</y>
      <width>111</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Изменить</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_6">
    <property name="geometry">
     <rect>
      <x>1290</x>
      <y>720</y>
      <width>161</width>
      <height>17</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Удалить человека №:</string>
    </property>
   </widget>
   <widget class="QWidget" name="">
    <property name="geometry">
     <rect>
      <x>1270</x>
      <y>90</y>
      <width>411</width>
      <height>410</height>
     </rect>
    </property>
    <layout class="QGridLayout" name="gridLayout">
     <item row="11" column="0">
      <widget class="QLabel" name="weightIntervalLabel">
       <property name="text">
        <string>Диапазон веса</string>
       </property>
      </widget>
     </item>
     <item row="9" column="2">
      <widget class="QSpinBox" name="ageMaxSpin">
       <property name="minimum">
        <number>18</number>
       </property>
       <property name="maximum">
        <number>120</number>
       </property>
       <property name="value">
        <number>120</number>
       </property>
      </widget>
     </item>
     <item row="8" column="0">
      <widget class="QLabel" name="patParamLabel">
       <property name="font">
        <font>
         <weight>75</weight>
         <bold>true</bold>
         <underline>false</underline>
        </font>
       </property>
       <property name="text">
        <string>Параметры партнёра</string>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="customLabel">
       <property name="font">
        <font>
         <pointsize>15</pointsize>
         <underline>true</underline>
        </font>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item row="9" column="1">
      <widget class="QSpinBox" name="ageMinSpin">
       <property name="minimum">
        <number>18</number>
       </property>
       <property name="maximum">
        <number>120</number>
       </property>
      </widget>
     </item>
     <item row="4" column="0">
      <widget class="QLabel" name="sexLabel">
       <property name="text">
        <string>Пол</string>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <widget class="QSpinBox" name="participantNumberSpin"/>
     </item>
     <item row="4" column="1">
      <widget class="QRadioButton" name="manRadioButton">
       <property name="text">
        <string>Мужской</string>
       </property>
      </widget>
     </item>
     <item row="2" column="0">
      <widget class="QLabel" name="participantNumberLabel">
       <property name="font">
        <font>
         <pointsize>12</pointsize>
        </font>
       </property>
       <property name="text">
        <string>Номер участника:</string>
       </property>
      </widget>
     </item>
     <item row="9" column="0">
      <widget class="QLabel" name="ageIntervalLabel">
       <property name="text">
        <string>Диапазон возраста</string>
       </property>
      </widget>
     </item>
     <item row="11" column="2">
      <widget class="QSpinBox" name="weightMaxSpin">
       <property name="maximum">
        <number>250</number>
       </property>
      </widget>
     </item>
     <item row="5" column="0">
      <widget class="QLabel" name="ageLabel">
       <property name="text">
        <string>Возраст</string>
       </property>
      </widget>
     </item>
     <item row="8" column="1">
      <widget class="QLabel" name="minLabel">
       <property name="font">
        <font>
         <weight>50</weight>
         <italic>true</italic>
         <bold>false</bold>
        </font>
       </property>
       <property name="text">
        <string>Минимум</string>
       </property>
      </widget>
     </item>
     <item row="3" column="0">
      <widget class="QLabel" name="fioLabel">
       <property name="text">
        <string>ФИО:</string>
       </property>
      </widget>
     </item>
     <item row="7" column="0">
      <widget class="QLabel" name="weightLabel">
       <property name="text">
        <string>Вес</string>
       </property>
      </widget>
     </item>
     <item row="10" column="1">
      <widget class="QSpinBox" name="highMinSpin">
       <property name="maximum">
        <number>250</number>
       </property>
      </widget>
     </item>
     <item row="3" column="1" colspan="2">
      <widget class="QTextEdit" name="fioEdit"/>
     </item>
     <item row="7" column="1">
      <widget class="QSpinBox" name="wheightSpin">
       <property name="maximum">
        <number>250</number>
       </property>
      </widget>
     </item>
     <item row="5" column="1">
      <widget class="QSpinBox" name="ageSpin">
       <property name="minimum">
        <number>18</number>
       </property>
       <property name="maximum">
        <number>120</number>
       </property>
      </widget>
     </item>
     <item row="4" column="2">
      <widget class="QRadioButton" name="womanRadioButton">
       <property name="text">
        <string>Женский</string>
       </property>
      </widget>
     </item>
     <item row="6" column="0">
      <widget class="QLabel" name="highLabel">
       <property name="text">
        <string>Рост</string>
       </property>
      </widget>
     </item>
     <item row="10" column="2">
      <widget class="QSpinBox" name="highMaxSpin">
       <property name="maximum">
        <number>250</number>
       </property>
      </widget>
     </item>
     <item row="11" column="1">
      <widget class="QSpinBox" name="weightMinSpin">
       <property name="maximum">
        <number>250</number>
       </property>
      </widget>
     </item>
     <item row="8" column="2">
      <widget class="QLabel" name="maxLabel">
       <property name="font">
        <font>
         <italic>true</italic>
        </font>
       </property>
       <property name="text">
        <string>Максимум</string>
       </property>
      </widget>
     </item>
     <item row="6" column="1">
      <widget class="QSpinBox" name="highSpin">
       <property name="maximum">
        <number>250</number>
       </property>
      </widget>
     </item>
     <item row="10" column="0">
      <widget class="QLabel" name="highIntervalLabel">
       <property name="text">
        <string>Диапазон роста</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QPushButton" name="addButton">
       <property name="text">
        <string>Добавить!</string>
       </property>
      </widget>
     </item>
     <item row="1" column="2">
      <widget class="QPushButton" name="changeButton">
       <property name="text">
        <string>Поменять!</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1730</width>
     <height>22</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu">
    <property name="title">
     <string>Файл</string>
    </property>
    <addaction name="OpenAction"/>
    <addaction name="SaveAction"/>
   </widget>
   <addaction name="menu"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="OpenAction">
   <property name="text">
    <string>Открыть</string>
   </property>
  </action>
  <action name="SaveAction">
   <property name="text">
    <string>Сохранить</string>
   </property>
   <property name="toolTip">
    <string>Сохранить</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>exitButton</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>1711</x>
     <y>759</y>
    </hint>
    <hint type="destinationlabel">
     <x>1755</x>
     <y>621</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
